### ISSUE TITLE
markdown.extension.preview.autoShowPreviewToSide

### ISSUE THREAD

**Issue by Tianzii** [yzhang-gh/vscode-markdown#1090]
## References

- **A new preview opens every time VSCode is reopened**

**Comment by yzhang-gh** [comment]
Don't understand what you want to express. What is the issue? What do you expect?

**Comment by Tianzii** [comment]
After turning on the "AutoShowPreviewToSide" option, each time the editor is opened, a preview tab will be opened repeatedly on the basis of the previous one.

**Comment by yzhang-gh** [comment]
Have you locked the preview tab (the title is `[Preview] ...` rather than `Preview ...`)? If it is the case, you can unlock it from the `···` menu on the top right corner.

**Comment by Lemmingh** [comment]
Yeah, VS Code's "Markdown Preview Locking" and "Lock Editor Group" feature can be causes.

Besides, could you please see if #144 reflects your problem?

Since I broke our CI last year unfortunately, there is currently no dev build to help diagnosis.

**Comment by Tianzii** [comment]
> 您是否锁定了预览选项卡（标题是而不是）？如果是这种情况，您可以从右上角的菜单中解锁它。`[Preview] ...``Preview ...``···`

Yes, that's why, thank you so much for saving us OCD sufferers (doge)

**Comment by Tianzii** [comment]
> 您是否锁定了预览选项卡（标题是而不是）？如果是这种情况，您可以从右上角的菜单中解锁它。`[Preview] ...``Preview ...``···`

Yes, that's why, thank you so much for saving us OCD sufferers (doge)

**Comment by Lemmingh** [comment]
你竟然一直在用机翻吗？

还是用汉语聊吧……
